custom_components/tuya_local/devices/cecotec_bigdry_4000_dehumidifier.yaml

```diff
@@ -0,0 +1,145 @@
+name: BigDry 4000 Expert
+products:
+  - id: d44nan1uokr1hlwf
+    name: Cecotec BigDry 4000 Expert
+    # {"dps": {"1": true, "3": "60", "4": "low", "6": 54, "7": 23, "19": 0}}
+primary_entity:
+  entity: humidifier
+  class: dehumidifier
+  name: Dehumidifier
+  dps:
+    - id: 1
+      name: switch
+      type: boolean
+    - id: 3
+      type: integer
```

Comment:
dp 3 above has quotes around it. This suggests that it is actually an string (enum) type, and will only accept a small number of specific values, not the full range between 40 and 80. The implementation below for a duplicate select entity seems to follow this, but to move it up here, you will need to remove the quotes from the `value` side of the mappings.